dsp-schema/core/src/main/scala/dsp/schema/domain/SchemaCommands.scala

```diff
@@ -0,0 +1,49 @@
+package dsp.schema.domain
+
+import dsp.errors.ValidationException
+import dsp.valueobjects.LangString
+import dsp.valueobjects.Schema
+import zio.prelude.Validation
+
+import java.time.Instant
+
+case class SmartIri(value: String)
+
+sealed abstract case class CreatePropertyCommand private (
```

Comment:
Now that we have the case class and make method, it would be good to test them with unit tests.

Reply:
done